api/src/main/java/org/openmrs/module/sync2/client/rest/impl/RestResourceConverterImpl.java

```diff
@@ -173,6 +173,17 @@ private void convertObservation(Map<String, Object> simpleObject) {
 		Map concept = (Map<String, Object>) simpleObject.get("concept");
 		simpleObject.remove("concept");
 		simpleObject.put("concept", concept.get("uuid"));
+
+		final String encounterKey = "encounter";
```

Comment:
Did you take a look at our naming conventions for constants?

Reply:
Being a constant whose value will not change, our convention for its name is here: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions